src/Leaflet.Editable.js

```diff
@@ -880,6 +898,162 @@ L.Editable.PolygonEditor = L.Editable.PathEditor.extend({
 
 });
 
+L.Editable.RectangleEditor = L.Editable.BaseEditor.extend({
+
+    initialize: function (map, feature, options) {
+        L.Editable.BaseEditor.prototype.initialize.call(this, map, feature, options);
+        this.anchorPoint = null;
+        this.vertexMarkers = null;
+        if (!this.isZeroRect()) {
+            this.anchorPoint = this.feature.getBounds().getNorthWest();
+            this.initVertexMarkers();
+        }
+    },
+
+    enable: function () {
+        if (this._enabled) return this;
+        L.Editable.BaseEditor.prototype.enable.call(this);
+        return this;
+    },
+
+    disable: function() {
+        return L.Editable.BaseEditor.prototype.disable.call(this);
+    },
+
+    initVertexMarkers: function () {
+        var points = this.getLatLngs();
+        this.vertexMarkers = points.map(function(p) {
+            return new this.tools.options.vertexMarkerClass(p, points, this);
+        }, this);
+    },
+
+    updateVertexMarkers: function(ignoreVertex, ignoreLatLng) {
+        var point = 0;
+        var marker = 0;
+        var points = this.getLatLngs();
+        while (point < 4 && marker < 4) {
+            if (ignoreLatLng && points[point].equals(ignoreLatLng)) {
+                point++;
+            } else if (this.vertexMarkers[marker] === ignoreVertex) {
+                marker++;
+            } else {
+                this.vertexMarkers[marker++].setLatLng(points[point++]);
+            }
+        }
+    },
+
+    addMiddleMarker: function (left, right, latlngs) {
```

Comment:
Maybe add a "withVertex" static property, that will then be checked by the parent method, instead